### DISCUSSION TITLE
Consistent look & feel

### DISCUSSION THREAD

**Discussion by malckier** [entel-me/entel#76] [Ideas]
I started looking through the issues and saw the one for [Polish Emails](https://github.com/entel-me/entel/issues/34). This got me thinking to where css could potentially be managed with Sass. I, unfortunately, am not great w/CSS. :) but I started looking at how it can be used to generate style sheets for the app and emails so that emails have a similar look to the app itself. I mention this to see what people think.

**Comment by antonykamp** [comment]
Hello, my friend,
Using the same style sheet for the app and emails sounds nice! But you wouldn't found any CSS-file in our app :/ Entel uses the component library chakra-UI with its integrated style sheets. Removing this library is a big task. But it's possible because chakra does nothing else than writing CSS :D But by removing it, we could use something like sass.

**Reply by malckier** [reply]
I suppose that could be lower priority for now, if you like, since chakra-UI is heavily integrated.

**Reply by antonykamp** [reply]
But we have to keep it in mind because chakra-UI isn't the best solution if we want to scale the application in the future.

**Reply by malckier** [reply]
Well, it's almost always easier to remove something you don't wish to keep at an earlier stage. I'm just not a great CSS'er

**Reply by antonykamp** [reply]
You are absolutely right 👍 ... I'll create a project board tomorrow with some big topics including the transformation from chakra-UI to SCSS.

**Comment by antonykamp** [comment]
Btw: It's difficult to include `.css` files in HTML-mail-templates. Such html-files allow style configurations in header only :/ 
Some packages exist which compile HTML, SCSS, and co. to a single HTML file... How does that sound?

(Please correct me if I'm wrong :)
